Makefile

```diff
@@ -194,6 +194,18 @@ antctl: $(ANTCTL_BINARIES)
 antctl-release:
 	@$(GO) build -o $(BINDIR)/$(ANTCTL_BINARY_NAME) $(GOFLAGS) -ldflags '-s -w $(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+.PHONY: check-copyright
+check-copyright: 
+	@echo "==> Installing addlicense <=="
+	@$(GO) install github.com/google/addlicense@latest
+ifneq ($(add), true)
+	@echo "===> Checking License for files <==="
+	$(CURDIR)/hack/add-license.sh 
+else
+	@echo "===> Adding License for files <==="
+	$(CURDIR)/hack/add-license.sh --add
```

Comment:
I feel it's unecessary to add `add-license.sh`, Could you just add the CLI here? we can already tell if it's `check` or `add` action in Makefile.